subprojects/docs/src/docs/release/notes.md

```diff
@@ -22,6 +22,37 @@ The new metadata format is still under active development, but it can already be
 
 [The JaCoCo plugin](userguide/jacoco_plugin.html) has been upgraded to use [JaCoCo version 0.8.0](http://www.jacoco.org/jacoco/trunk/doc/changes.html) by default.
 
+### Public API for defining command line options for tasks
```

Comment:
In general, I really like the level of detail and example for this bit of the release notes.

Reply:
Thanks!